src/molecules/Modal/ModalHeader.native.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+import Space from '../../atoms/Space';
+import Size from '../../atoms/Size';
+import Flex from '../../atoms/Flex';
+import View from '../../atoms/View';
+import Button from '../../atoms/Button';
+
+const FullScreenModalHeader = styled(View)`
+  background-color: ${(props) => props.theme.colors.background[200]};
+  shadow-color: ${(props) => props.theme.colors.primary[930]};
+  shadow-offset: 0px 4px;
+  shadow-opacity: 1;
+  shadow-radius: 15px;
+  elevation: 2;
+`;
+
+const Divider = styled(View)`
+  background-color: ${(props) => props.theme.colors.shade[920]};
+`;
+
+const ModalHeader = ({ children, type, onClose }) => {
+  return type === 'centered' ? (
```

Comment:
why can't you put conditions here and return from that instead of so many tertiary operators?